style.css

```diff
@@ -837,19 +829,25 @@ body{
     row-gap: 0px;
 }
 .footer-secondpart-grid-item{
+    /* display: grid;
```

Comment:
Please change the names of footer-secondpart to something like footer-midsection.